src/style.css

```
@@ -1,3 +1,88 @@
+:root {
+  --active-color: #00acdc;
+ }
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  width: 100%;
+}
+
+
+.header {
```

```suggestion
}

.header {
```

The same in other places